### PR TITLE
[v9.2.x] Revert: Query editor components from grafana-ui

### PR DESCRIPTION
**Pull request by sunker**
**What this PR does / why we need it**:
Manual backport of https://github.com/grafana/grafana/pull/57436

### CODE REVIEW

File: public/app/plugins/datasource/mssql/sqlCompletionProvider.ts [hunk 1]
```diff
@@ -35,13 +36,17 @@ export const getSqlCompletionProvider: (args: CompletionProviderGetterArgs) => L
...
 
-        return tablePath;
+        return { table: tablePath };
       },
     },
 
     columns: {
-      resolve: async (t: string) => {
-        const [database, schema, tableName] = t.split('.');
+      resolve: async (t: TableIdentifier | undefined) => {
```
Comment (jackw) [inline public/app/plugins/datasource/mssql/sqlCompletionProvider.ts:44]: This seems to be the only change in the backport that changes code beyond an import. I don't know enough about the mssql datasource but should these changes be backported?

**Review by jackw — APPROVED** [review]
LGTM! 🚀 

Left a comment regarding a change that appears to be more than import paths.

File: public/app/plugins/datasource/mssql/sqlCompletionProvider.ts (hunk 1 above)
Comment (sunker) [reply public/app/plugins/datasource/mssql/sqlCompletionProvider.ts:44]: @scottlepp WDYT?

Comment (zoltanbedi) [reply public/app/plugins/datasource/mssql/sqlCompletionProvider.ts:44]: This should be okay.

**Pull request merged**